modules/dxp/apps/commerce-avalara/commerce-avalara-tax-engine-fixed-web/src/main/java/com/liferay/commerce/avalara/tax/engine/fixed/web/internal/servlet/taglib/ui/CommerceTaxMethodAvalaraScreenNavigationEntry.java

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.commerce.avalara.tax.engine.fixed.web.internal.servlet.taglib.ui;
+
+import com.liferay.commerce.avalara.tax.engine.fixed.configuration.CommerceTaxAvalaraTypeConfiguration;
+import com.liferay.commerce.constants.CommerceTaxScreenNavigationConstants;
+import com.liferay.commerce.currency.service.CommerceCurrencyLocalService;
+import com.liferay.commerce.percentage.PercentageFormatter;
+import com.liferay.commerce.product.model.CommerceChannel;
+import com.liferay.commerce.product.service.CPTaxCategoryService;
+import com.liferay.commerce.tax.model.CommerceTaxMethod;
+import com.liferay.commerce.tax.service.CommerceTaxMethodService;
+import com.liferay.frontend.taglib.servlet.taglib.ScreenNavigationCategory;
+import com.liferay.frontend.taglib.servlet.taglib.ScreenNavigationEntry;
+import com.liferay.frontend.taglib.servlet.taglib.util.JSPRenderer;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.module.configuration.ConfigurationProvider;
+import com.liferay.portal.kernel.security.permission.resource.ModelResourcePermission;
+import com.liferay.portal.kernel.settings.GroupServiceSettingsLocator;
+import com.liferay.portal.kernel.settings.ParameterMapSettingsLocator;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.portal.kernel.util.ResourceBundleUtil;
+
+import java.io.IOException;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Calvin Keum
+ */
+@Component(
+	enabled = true,
+	property = {
+		"screen.navigation.category.order:Integer=20",
+		"screen.navigation.entry.order:Integer=10"
+	},
+	service = {ScreenNavigationCategory.class, ScreenNavigationEntry.class}
+)
+public class CommerceTaxMethodAvalaraScreenNavigationEntry
```

Comment:
` Name of class implementing 'ScreenNavigationCategory' should end with 'ScreenNavigationCategory': ./modules/dxp/apps/commerce-avalara/commerce-avalara-tax-engine-fixed-web/src/main/java/com/liferay/commerce/avalara/tax/engine/fixed/web/internal/servlet/taglib/ui/CommerceTaxMethodAvalaraScreenNavigationEntry.java (SourceCheck:JavaClassNameCheck)`

Gotta rename this to `CommerceTaxMethodAvalaraScreenNavigationCategory`, its a SF change that was not on 7.1. We currently have the check suppressed on the other Commerce modules, but we should start following the rules here.

Reply:
Got it, will update and fix. Thanks Danny.